### ISSUE TITLE
Stone generating in wrong place

### ISSUE THREAD

**Issue by sarim** [TreeboMC/StoneOres2#4]
config used for testing
```yml
  world:
    checkIslandLevel: 'false'
    lvlcommand: cb level
    tiers:
      default: 0
    blocktypes:
      default:
        DIAMOND_ORE: 1    
```

I'm testing with paper version #60 with no other plugin. Here is the default lava and water interaction. https://gfycat.com/wiltedflamboyantanaconda

But when using stoneores2, it generates the ore in wrong position. Also see it fails to capture the event in bottom [ where a stone is generated ]
https://gfycat.com/glamorousecstaticarrowworm

The outcome of water and lava is various. But reading the code i'm not seeing the algorithm for other cases. But thats not really the issue. The GetPosition part of the code is returning wrong value.

wiki: https://minecraft.gamepedia.com/Liquid#Mixing

**Comment by ShakeforProtein** [comment]
Fixed in version 0.1.4

**Comment by sarim** [comment]
Thanks. Now it seems to be working fine.
